src/cloudant/design_document.py

```diff
@@ -243,3 +207,128 @@ def info(self):
         GET databasename/_design/{ddoc}/_info
         """
         raise NotImplementedError("_info not yet implemented")
+
+class CloudantDesignDocument(DesignDocument):
+    """
+    Encapsulates a Cloudant version of a
+    :class:`~cloudant.design_document.DesignDocument`.
+    A CloudantDesignDocument object is instantiated with a reference
+    to a Cloudant database and provides an API to query index management,
+    list and show functions, etc.
+
+    Note:  Currently only the query view management API exists.  Remaining
+    Cloudant design document functionality will be added later.
+
+    :param database: A ``CloudantDatabase`` instance used by the
```

Comment:
This description definitely needs more clarity.  See [this comment](https://github.com/cloudant/python-cloudant/pull/168/files#r62885465) and this [other comment](https://github.com/cloudant/python-cloudant/pull/168/files#r62886306).  Be sure that these two cases (DesignDocument and CloudantDesignDocument) are handled similarly.